source/assets/stylesheets/styles/_contact_page_address.scss

```diff
@@ -0,0 +1,70 @@
+.contact-page-address {
+  line-height: 1.4em;
+}
+
+.contact-page-address__phone-number {
+  padding: 20px 0 15px;
+  font-weight: 300;
+
+  a {
+    color: $dark-gray;
+    text-decoration: none;
+  }
+}
+
+.contact-page-address__email {
+  padding-bottom: 15px;
+  font-weight: 300;
+
+  a {
+    color: $dark-gray;
+    text-decoration: none;
+  }
+}
+
+.contact-page-address__map-link {
+  padding-bottom: 20px;
+  font-weight: 300;
+  border-bottom: solid 1px lighten($medium-gray, 30%);
+
+  a {
+    color: lighten($medium-gray, 20%);
+    text-decoration: none;
+  }
+}
+
+i.contact-page-address__phone-icon {
+  color: $medium-gray;
+  padding-right: 5px;
+}
+
+i.contact-page-address__email-icon {
+  color: $medium-gray;
+  padding-right: 4px;
+}
+
+i.contact-page-address__map-icon {
+  color: $medium-gray;
+  padding-right: 5px;
+}
+
+.contact-page-address__directions {
+  padding: 15px 0 20px;
+}
+
+.contact-page-address__directions-station {
+  padding-top: 15px;
+
+  a {
+    color: lighten($medium-gray, 20%);
+    text-decoration: none;
+  }
+}
+
+.contact-page-address__transport-icon {
+  display: inline-block;
+  height: 17px;
+  padding-right: 8px;
+  width: 20px;
+  vertical-align: middle;
```

Comment:
aaa definitely need to install that plugin to rubymine, constantly keep forgetting about alphabetical order